buildspec.yml

```diff
@@ -1,11 +1,29 @@
 version: 0.2
 
+env:
+  variables:
+    SMTP_CREDS_SECRET: cfpb/team/regtech/smtp-ses-creds
+    SMTP_FROM_ADDRESS: no-reply@cfpb.gov
+  secrets-manager:
+    TL_CONSOLE_URL: cfpb/team/regtech/twistlock:TL_CONSOLE_URL
+    TL_USER: cfpb/team/regtech/twistlock:TL_USER
+    TL_PASSWORD: cfpb/team/regtech/twistlock:TL_PASSWORD
+    SMTP_USERNAME: "${SMTP_CREDS_SECRET}:username"
+    SMTP_PASSWORD: "${SMTP_CREDS_SECRET}:password"
+    SMTP_SERVER: "${SMTP_CREDS_SECRET}:smtp_server"
+    EMAIL_LIST: "${SMTP_CREDS_SECRET}:email_list"
+
 phases:
   pre_build:
     commands:
       # Set envvars dependent on CodeBuild project's own envvars
+      - sudo yum -y install mailx coreutils --allowerasing --skip-broken
+      - export JOB_NAME=$CODEBUILD_BUILD_ID
+      - export IMAGE_NAME="cfpb/${TEAM_NAMESPACE}/sbl-frontend"
       - export IMAGE_NAME="cfpb/${TEAM_NAMESPACE}/sbl-frontend"
-      - export IMAGE_TAG="preview"
+      - export IMAGE_TAG="twistlock-cli-codebuild"
```

Comment:
Are we going to switch this back before merge?

Reply:
Good catch, thanks. Updated.